createcourse/tasks.py

```diff
@@ -16,10 +14,16 @@
 from .api_settings import workshop_url, yaksh_post_url, workshop_json_file_mapper
 import os
 
+from django_celery_results.models import TaskResult
 
 @celery.task
 def fetch_data():
     params = {'status': 1}
+    last_fetched_task=TaskResult.objects.order_by('-date_done')
+    if len(last_fetched_task):
+        params['last_fetched']=last_fetched_task[0].date_done.strftime('%Y-%m-%d')
+    else:
+        params['last_fetched']=datetime.date.today().strftime('%Y-%m-%d')
```

Comment:
I guess this is not `last_fetched` in the workshop API while getting workshops. It is 'from_date'. Check that and test it.

Reply:
Ok sir... I will go through it.